src/python/run_matching.py

```diff
@@ -51,11 +51,50 @@ def setDictParam(args):
     params["nonMatchCost"] = 3.7
     params["bufferSize"] = 100
 
-    queriesNum = len(list(args.query_features.glob("*Feature.pb")))
+    queriesNum = len(list(query_features_dir.glob("*Feature.pb")))
     params["querySize"] = queriesNum
     return params
 
 
+def computeFeatures(image_dir, output_folder, feature_name_prefix):
+    if output_folder.exists():
+        print(
+            "WARNING: Feature folder {folder_name} exists. Skipping feature computation.".format(
+                folder_name=output_folder
+            )
+        )
+        return
+    output_folder.mkdir()
+    # Extract features.
+    netvlad_weights = Path("netvlad/data/Pitts30K_struct.mat")
+    if not netvlad_weights.exists():
+        print(
+            "ERROR: Can't find netvlad weights. Please download to netvlad/data/ from https://cvg-data.inf.ethz.ch/hloc/netvlad/Pitts30K_struct.mat"
```

Comment:
We might want to put this file into git LFS of this repository to avoid the need of downloading it manually.

Reply:
Thanks for the suggestion. Maybe some other time. Maybe when some user will actually need this :smiling_face_with_tear: